litex/tools/litex_json2dts.py

```diff
@@ -32,14 +32,21 @@ def generate_dts(d):
 """
 
     # Boot Arguments -------------------------------------------------------------------------------
-    linux_initrd_start_offset = {
+    default_initrd_start_offset = {
         "mor1kx":               8*mB,
         "vexriscv smp-linux" : 16*mB,
     }
-    linux_initrd_end_offset = {
+    default_initrd_end_offset = {
         "mor1kx":              16*mB,
         "vexriscv smp-linux" : 24*mB,
     }
+    if initrd_start_offset is None:
+        initrd_start_offset = default_initrd_start_offset[cpu_name]
+
+    if initrd_size is None:
+        initrd_end_offset = default_initrd_end_offset[cpu_name]
+    else:
+        initrd_end_offset = initrd_start_offset + initrd_size
```

Comment:
This is a low-prio, but we are mixing `size` and `end_offset` which might lead to some confusion. 

Wouldn't it be better to change `default_initrd_end_offset` to `default_initrd_size`?
In fact from what I see `default_initrd_size` seems to be 8mB for both cpus, so maybe a dictionary is not needed at all?

Reply:
Yeah, I was just trying to keep the structure the same.  I will change it.

Reply:
Fix, this too, removing the second dictionary make it much cleaner now.